doc/source/whatsnew/v3.0.0.rst

```diff
@@ -24,6 +24,12 @@ Enhancement1
 Enhancement2
 ^^^^^^^^^^^^
 
+New Deprecation Policy
+^^^^^^^^^^^^^^^^^^^^^^
+pandas 3.0.0 introduces a new 3-stage deprecation policy: using ``DeprecationWarning`` initially, then switching to ``FutureWarning`` for broader visibility in the last minor version before the next major release, and then removal of the deprecated functionality in the major release.
```

Comment:
Shouldn't we say that these could be tested via the classes `PandasDeprecationWarning` and `PandasFutureWarning` ?